CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Next Release
 
 * [#269](https://github.com/intridea/hashie/pull/272): Added Hashie::Extensions::DeepLocate - [@msievers](https://github.com/msievers).
+* [#270](https://github.com/intridea/hashie/pull/277): Fixed crash with IndifferentAccess and HashWithIndifferentAccess - [@gardenofwine](https://github.com/gardenofwine).
```

Comment:
This should be something other than "crash", maybe "Fix: ArgumentException raised when ..."